js/extension/components/PlotSelection.jsx

```diff
@@ -57,58 +57,72 @@ function PlotSelectionTabContent({
     );
 }
 
-function PlotSelectionTabActionButtons({onNewTab = () => {}, onTabDelete = () => {}}) {
+function PlotSelectionTabActionButtons({onNewTab = () => {}}) {
     return (
-        <ButtonGroup className="pull-right">
+        <ButtonGroup className="plotSelectionTabActionButtons">
             <OverlayTrigger placement="bottom" overlay={<Tooltip><Message msgId={'cadastrapp.search.addTab'}/></Tooltip>}>
                 <Button
-                    className="pull-right"
                     onClick={onNewTab}
                 ><span className="glyphicon glyphicon-plus"></span>
                 </Button>
             </OverlayTrigger>
-            <OverlayTrigger placement="bottom" overlay={<Tooltip><Message msgId={'cadastrapp.search.deleteTab'}/></Tooltip>}>
-                <ConfirmButton
-                    className="pull-right"
-                    confirmContent={<Message msgId={'cadastrapp.search.confirmDeleteTab'}/>}
-                    onClick={onTabDelete}>
-                    <Glyphicon glyph="trash" />
-                </ConfirmButton>
-            </OverlayTrigger>
         </ButtonGroup>
     );
 }
 
+function DeletePlot ({
+    index, 
+    isDrop=false,
+    onTabDelete,
+    MAX_TABS
+    }) {    
+    return (
+        <OverlayTrigger placement="bottom" overlay={<Tooltip><Message msgId={'cadastrapp.search.deleteTab'}/></Tooltip>}>
+            <ConfirmButton
+                href="javascript:void(0)"
+                confirmContent={<Message msgId={'cadastrapp.search.confirmDeleteTab'}/>}
+                onClick={(e) => {
+                    e.stopPropagation();
+                    onTabDelete(!isDrop ? index : index + (MAX_TABS - 1));
```

Comment:
This func needs a minor adjustment.

Indeed, the index calculation should be out of this function.
It should be directly read from props.

Call component  :

`<DeletePlot index={index + (MAX_TABS - 1)} isDrop={true} onTabDelete={onTabDelete}/>`

Component function will just read `index` prop :

```
function DeletePlot({isDrop, onTabDelete, index}) {
...
    onClick={(e) => {
        e.stopPropagation();
        onTabDelete(index);
     }}
...
}
```

Reply:
You can also just pass the delete func as property to reduce function complexity:

`<DeletePlot delete={() => onTabDelete(index + (MAX_TABS - 1))}/>`


Function : 

```
function DeletePlot ({
    delete = () => {},
}) {    
    return (
        <OverlayTrigger placement="bottom" overlay={<Tooltip><Message msgId={'cadastrapp.search.deleteTab'}/></Tooltip>}>
            <ConfirmButton
                href="javascript:void(0)"
                confirmContent={<Message msgId={'cadastrapp.search.confirmDeleteTab'}/>}
                onClick={(e) => {
                    e.stopPropagation();
                    delete();
                    }}>
                <Glyphicon glyph="remove" />
            </ConfirmButton>
        </OverlayTrigger>
    )
}
```